velox/vector/DictionaryVector.h

```diff
@@ -80,6 +80,19 @@ class DictionaryVector : public SimpleVector<T> {
 
   bool isNullAt(vector_size_t idx) const override;
 
+  bool containsNullAt(vector_size_t idx) const override {
+    if constexpr (std::is_same_v<T, ComplexType>) {
```

Comment:
not sure I understand this part. In which cases we have dictionaries with ComplexType and when we don't?

Reply:
A dictionary vector can wrap a primitive type vector or a complex type vector. In the former case T will be primitive type. In the latter case T will be ComplexType.